### PR TITLE
We're back!

### PR DESCRIPTION
**Pull request by KyleMit**

### CODE REVIEW

File: src/index.njk [hunk 1]
```diff
@@ -23,25 +22,35 @@ meta_description: Vermont Code Camp is going to be on hiatus for 2021.
             </a>
         {% endif %}
 
+        <p>We've missed seeing you these past few years and hope you haven't forgotten about the Vermont Code Camp. As you know, we've been on hiatus during the COVID pandemic. We sincerely hope you and your families have survived the last couple of years.</p>
+        <p>We are happy to announce the 2022 return of Vermont Code Camp!</p>
+        <p>On September 24 we will return to Champlain College who has generously opened their doors to host us again.</p>
```
Comment (robhalevt) [inline src/index.njk]: Please add link to Champlain College's website
```suggestion
        <p>On September 24 we will return to [Champlain College](https://www.champlain.edu/) who has generously opened their doors to host us again.</p>
```

**Review by robhalevt — CHANGES_REQUESTED** [review]
Just need to add a link to Champlain College

File: src/index.njk (hunk 1 above)
Comment (KyleMit) [reply src/index.njk]: good catch - don't think markup will here so discarding the suggestion, but added an anchor tag

**Review by robhalevt — APPROVED** [review]

**Comment by zekefarwell** [comment]
Thanks for handling this, @KyleMit.  Glad the deploy process is working as expected despite getting little to no attention over the past couple years!

**Pull request merged**